Ge02 team6

lare-odoo opened the pull request:
PR from GE02 to stg-dev

cogs-odoo committed:
created new module folder

lare-odoo committed:
[ADD] motorcycle_registry: added map view to motorcycles
The motorcycle registry views were modified to add a new map view to show the locations of the motorcycles in the registry. This was done by inheriting the motorcycle view modes and adding 'map', as opposed to the initial idea of inheriting directly from the "project_enterprise" module

yall-odoo commented on GE02-TEAM6/__init__.py:
```diff
@@ -0,0 +1 @@
+from . import models
```
new line missing

yall-odoo commented on GE02-TEAM6/__manifest__.py:
```diff
@@ -1,17 +1,22 @@
\ No newline at end of file
+    'demo' : [],
+    "application": False,
+    "license": "OPL-1",
```
Use either `'var'` or `"var"`, not both.

yall-odoo reviewed (changes requested):
Formatting concerns but other than that, beautiful.

lare-odoo committed:
[IMP] Better formatting of files
Added an extra line break at the end of the file "__init__.py", and changed all double quotes to single quotes in the file "__manifest__.py"

yall-odoo reviewed (approved):
Awesome.
